MonkeyWrench.Web.UI/ViewLane.aspx.cs

```diff
@@ -133,19 +195,20 @@ public string GenerateLane (GetViewLaneDataResponse response)
 		DBLane lane = response.Lane;
 		DBHost host = response.Host;
 		DBRevision revision = response.Revision;
+		bool hidden = IsBranchProtected(lane) || revisionwork.State == DBState.Success;

```

Comment:
The success check is not needed, because below we only add the links if the state is `Executing`, `Aborted` or `Issues & not completed`.

Also please move the check below to make sure it's only executed if necessary:

    if (isExecuting) {
        var hidden = IsBranchProtected (lane):
        ....
    }